docs/get-started.md

```diff
@@ -46,8 +50,51 @@ cd -
 gittuf version
 ```
 
-**Building from source.** To build from source, clone the repository and run
+### Windows
+
+Run this script as a `.ps1` file (PowerShell script).
+
+```powershell
+# Modify these values as necessary.
+# One of: amd64, arm64
+ARCH="amd64"
+OS="windows"
+# See https://github.com/gittuf/gittuf/releases for the latest version
+VERSION="0.5.2"
+
+Push-Location # this saves the current working directory to the stack
+
+Invoke-WebRequest -Uri "https://github.com/gittuf/gittuf/releases/download/v$VERSION/gittuf_${VERSION}_${OS}_${ARCH}.exe" -OutFile "gittuf_${VERSION}${OS}${ARCH}.exe"
+Invoke-WebRequest -Uri "https://github.com/gittuf/gittuf/releases/download/v$VERSION/gittuf_${VERSION}_${OS}_${ARCH}.exe.sig" -OutFile "gittuf_${VERSION}${OS}${ARCH}.exe.sig"
+Invoke-WebRequest -Uri "https://github.com/gittuf/gittuf/releases/download/v$VERSION/gittuf_${VERSION}_${OS}_${ARCH}.exe.pem" -OutFile "gittuf_${VERSION}${OS}${ARCH}.exe.pem"
```

Comment:
Okay, they can be made to use `curl`. However, now that I think of it: for this "script" to run in command prompt, we'll have to provide a .exe file as well since .ps1 files won't run in command prompt. The use case where people will use `curl` on command prompt is if they manually type it in.